sdk/harambe/types.py

```diff
@@ -71,4 +71,5 @@ class HarnessOptions(TypedDict, total=False):
     disable_go_to_url: bool
     on_start: Optional[Callback]
     on_end: Optional[Callback]
+    on_new_page: Optional[Callback]
```

Comment:
can you make this of type Optional[PageCallback]? 

    on_start: Optional[Callback]
    on_end: Optional[Callback]
    
    both expect ctx -> awaitable not page -> awaitable